02-peer-protocol.md

```diff
@@ -1256,6 +1256,10 @@ A node:
   the last `revoke_and_ack` the receiving node sent, AND the receiving node
   hasn't already received a `closing_signed`:
     - MUST re-send the `revoke_and_ack`.
+    - if it has previously sent a `commitment_signed` that needs to be
+    retransmitted:
+      - MUST retransmit `revoke_and_ack` and `commitment_signed` in the same
```

Comment:
Perhaps we should also spell out the _recommended_ persistence requirements here as well? This was something `lnd` did incorrectly in a few places (not storing what we need to in order to properly reconstruct a commitment to validate a retransmission. However np with also leaving this off for a more fundamental path to make things more explicit generally.

Reply:
Ah re-reading again, there _is_ a light requirement re naive persistence down below where the diff touches the spec.

Reply:
Yes, I didn't update that paragraph about persistence that rusty wrote a while ago, but I could, let me know what changes you think would make sense there (if any)!